src/Appwrite/Platform/Modules/Sites/Http/Deployments/Template/Create.php

```diff
@@ -208,6 +196,8 @@ public function action(
             ]))
         );
 
+        $this->updateManualRuleForNewSiteDeployment($project, $site, $deployment, $dbForPlatform);
```

Comment:
Sounds a bit too long. Maybe `->updateEmptyRules()`? Maybe something else. But I think we need to simplify it a bit